src/main/scala/com/microsoft/sqlserver/jdbc/spark/SQLServerBulkJdbcOptions.scala

```diff
@@ -1,59 +1,78 @@
 package com.microsoft.sqlserver.jdbc.spark
 
 import java.sql.Connection
+
 import org.apache.spark.sql.catalyst.util.CaseInsensitiveMap
-import org.apache.spark.sql.execution.datasources.jdbc.JdbcOptionsInWrite
-
-class SQLServerBulkJdbcOptions(val params: CaseInsensitiveMap[String]) extends JdbcOptionsInWrite(params) {
-
-    def this(params: Map[String, String]) = this(CaseInsensitiveMap(params))
-    
-    // Save original parameters for when a JdbcBulkOptions instance is passed
-    // from the Spark driver to an executor, which loses the reference to the
-    // params input in memory
-    override val parameters = params
-
-    val user = params.getOrElse("user", null)
-    val password = params.getOrElse("password", null)
-    val dbtable = params.getOrElse("dbtable", null)
-
-    // If no value is provided, then we write to a single SQL Server instance.
-    // A non-empty value indicates the name of a data source whose location is 
-    // the data pool that the user wants to write to. This data source will
-    // contain the user's external table.
-    val dataPoolDataSource = params.getOrElse("dataPoolDataSource", null)
-
-    // In the standard Spark JDBC implementation, the default isolation level is
-    // "READ_UNCOMMITTED," but for SQL Server, the default is "READ_COMMITTED"
-    override val isolationLevel = params.getOrElse("mssqlIsolationLevel", "READ_COMMITTED") match {
-        case "READ_UNCOMMITTED" => Connection.TRANSACTION_READ_UNCOMMITTED
-        case "READ_COMMITTED"   => Connection.TRANSACTION_READ_COMMITTED
-        case "REPEATABLE_READ"  => Connection.TRANSACTION_REPEATABLE_READ
-        case "SERIALIZABLE"     => Connection.TRANSACTION_SERIALIZABLE
-        case "SNAPSHOT"         => Connection.TRANSACTION_READ_COMMITTED + 4094
+import org.apache.spark.sql.execution.datasources.jdbc.{
+  JDBCOptions,
+  JdbcOptionsInWrite
+}
+
+class SQLServerBulkJdbcOptions(val params: CaseInsensitiveMap[String])
+    extends JdbcOptionsInWrite(params) {
+
+  def this(params: Map[String, String]) = this(CaseInsensitiveMap(params))
+
+  // Save original parameters for when a JdbcBulkOptions instance is passed
+  // from the Spark driver to an executor, which loses the reference to the
+  // params input in memory
+  override val parameters = params
+
+  val dbtable = params.getOrElse("dbtable", null)
+  val databaseName = params.getOrElse("databaseName", null)
+
+  val user = params.getOrElse("user", null)
+  val password = params.getOrElse("password", null)
+
+  val accessToken = params.getOrElse("accessToken", null)
+  val encrypt = params.getOrElse("encrypt", null)
+  val hostNameInCertificate = params.getOrElse("hostNameInCertificate", null)
```

Comment:
encrypt and hostNameInCertificate should be pass-through options to JDBC driver, why do we need to define it here?

Reply:
I was mainly following suit with the fact other parameters were defined here that may be pass-thru as well.  User/password are examples of options we don't use in the connector but explicitly define.  I was assuming there was a reason behind it.

Going to remove them and re-test to see what happens...

Reply:
@shanyu
It is, in fact, pass-thru and does not need to be specified. I'll remove the references from my PR and revise my tests. I'd ask ideally to clean up the existing ones in a separate PR.